debug/process/stateExport.go

```diff
@@ -0,0 +1,169 @@
+package process
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go/common"
+	"github.com/ElrondNetwork/elrond-go/state"
+)
+
+// ExportUserAccountState will try to export the account state of a provided address
+func ExportUserAccountState(accountsDB state.AccountsAdapter, identifier string, address []byte, parentDirForFiles string) {
+	code, csvHexedData, err := getCodeAndData(accountsDB, address)
+	if err != nil {
+		log.Error("ExportUserAccountState", "identifier", identifier, "error", err)
+		return
+	}
+
+	err = exportCode(identifier, code, parentDirForFiles)
+	if err != nil {
+		log.Error("ExportUserAccountState", "identifier", identifier, "error", err)
+		return
+	}
+
+	err = exportData(identifier, csvHexedData, parentDirForFiles)
+	if err != nil {
+		log.Error("ExportUserAccountState", "identifier", identifier, "error", err)
+		return
+	}
+}
+
+func getCodeAndData(accountsDB state.AccountsAdapter, address []byte) (code []byte, csvHexedData []string, err error) {
+	account, err := accountsDB.GetExistingAccount(address)
+	if err != nil {
+		return nil, nil, fmt.Errorf("%w while getting existing data for hex address %s",
+			err, hex.EncodeToString(code))
```

Comment:
wrong print. `address` instead of `code`

Reply:
changed